roles/satellite-clone/tasks/main.yml

```diff
@@ -86,19 +86,27 @@
   command: yum clean all
 
 # turn off firewall
-- name: turn off firewalld - rhel7
-  service: name=firewalld enabled=no state=stopped
-  when: ansible_distribution_major_version == "7" and disable_firewall
+- name: turn off firewalld
+  service:
+    name: firewalld
+    enabled: no
+    state: stopped
+  when: disable_firewall
 
 # Update hostname
 - name: set host_name
-  hostname: name={{ hostname }}
+  hostname:
+    name: "{{ hostname }}"
+
 - name: check /etc/hostname
```

Comment:
What is the outcome here? Making the change in this PR or separately?

Reply:
I've now re-read this thread, and am not sure anymore what we wanted to get to here.
The line the comment is attached to is talking about `/etc/hostname`, which should have been updated by the `hostname` module in the step before.

We then started talking about `/etc/hosts`, which is a completely different story. And is only handled in line 100 below (and I didn't touch that, besides reformating).

If you want, I can add back the `ansible_distribution_major_version == "7"` to this step and thus restore the original behaviour, and we re-visit this at a later point?

Reply:
> If you want, I can add back the `ansible_distribution_major_version == "7"` to this step and thus restore the original behaviour, and we re-visit this at a later point?

I think that was my point: I don't think this `fileinline` is needed (nor desired) on EL8.

Reply:
> I think that was my point: I don't think this `fileinline` is needed (nor desired) on EL8.

Strictly speaking, it shouldn't be needed on EL7 either, but this step is there since the beginning of the repo and I am not sure why it was added.

Reply:
But I've added it back, so we can discuss/remove later ;)

Reply:
My reasoning here was to keep EL8 clean and see if we then still need to deal with EL7